### PR TITLE
Switch header for registering blocks

### PR DESCRIPTION
**Pull request by zhen0**
### Example

I was looking for how to register a block in orion and skipped over this because the title said "in Prefect Cloud".  This small change makes it clearer that you can also register a block for the orion ui.

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a `fix`, `feature`, `enhancement`, `docs`, `collections`, or `maintenance` label categorizing the change.

### CODE REVIEW

File: docs/concepts/blocks.md [hunk 1]
```diff
@@ -249,7 +249,7 @@ my_s3_bucket.save("my_s3_bucket")
 
 In the above example, the values for `AWSCredentials` are saved with `my_s3_bucket` and will not be usable with any other blocks.
 
-## Registering blocks for use in Prefect Cloud
+## Registering blocks for use in the Prefect UI:
```
Comment (tpdorsey) [inline docs/concepts/blocks.md:252]:
```suggestion
## Registering blocks for use in the Prefect UI
```

**Review by tpdorsey — APPROVED** [review]
Thank you!

**Comment by zhen0** [comment]
Hmmm.... any ideas what's breaking tests? If not, I can take a look later today/Monday.

**Comment by tpdorsey** [comment]
@zhen0 I'm taking a look to see if I can replicate the netlify errors locally

**Pull request merged**